mlrun/serving/states.py

```diff
@@ -1291,6 +1291,15 @@ def _add_graphviz_flow(
                 else {}
             )
             graph.edge(previous_object.fullname, child.fullname, **kw)
+        before = getattr(child, "before", [])
+        for item in before:
+            next_object = step[item]
+            kw = (
+                {"ltail": "cluster_" + next_object.fullname}
+                if next_object.kind == StepKinds.router
+                else {}
+            )
```

Comment:
Small thing, but I think this is a bit clearer, and makes it easier to add keywords in the future
```suggestion
            kw = {}
            if next_object.kind == StepKinds.router:
                kw["ltail"] = f"cluster_{next_object.fullname}"
```

Reply:
fixed and uploaded a notebook